papaya-web/src/main/java/pl/edu/agh/papaya/mappers/Mapper.java

```diff
@@ -0,0 +1,15 @@
+package pl.edu.agh.papaya.mappers;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public abstract class Mapper<M, A> {
```

Comment:
a może interfejs?

Reply:
W sensie masz za tym jakieś argumenty czy Ty byś miał instynkt, żeby zrobić interfejs?

Reply:
no jeśli nie ma powodu żeby zrobić klasę abstrakcyjną to ja robię interfejs